dts/riscv/bouffalolab/bl60x.dtsi

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2021-2025 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <freq.h>
+#include <mem.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			device_type = "cpu";
+			compatible = "sifive,e24", "riscv";
+			reg = <0>;
+			riscv,isa = "rv32imafcb";
+			hardware-exec-breakpoint-count = <4>;
+			status = "okay";
+
+			ictrl: interrupt-controller {
+				#address-cells = <0>;
+				#interrupt-cells = <1>;
+				compatible = "riscv,cpu-intc";
+				interrupt-controller;
+			};
+		};
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		ranges;
+
+		itcm: itcm@22010000 {
+			compatible = "zephyr,memory-region", "sifive,dtim0";
+			reg = <0x22010000 DT_SIZE_K(16)>;
+			zephyr,memory-region = "ITCM";
+		};
+
+		dtcm: dtcm@42014000 {
+			compatible = "zephyr,memory-region", "sifive,dtim0";
+			reg = <0x42014000 DT_SIZE_K(48)>;
+			zephyr,memory-region = "DTCM";
+		};
+
+		sram0: memory@42020000 {
+			compatible = "mmio-sram";
+			reg = <0x42020000 DT_SIZE_K(64)>;
+		};
+
+		sram1: memory@42030000 {
+			compatible = "mmio-sram";
+			reg = <0x42030000 DT_SIZE_K(112)>;
+		};
+
+		clic: clic@2000000 {
+			compatible = "sifive,clint0";
+			reg = <0x2000000 0x10000>;
+			reg-names = "control";
+
```

Comment:
inconsistent spacing in file

Reply:
I don't see any inconsistency. This is from current devicetree spec v0.4.

![image](https://github.com/user-attachments/assets/8576270d-7fd2-4601-be15-fc8aacf25510)

I assume that if it is on the documentation example is correct.